src/sim.rs

```diff
@@ -192,6 +204,13 @@ impl<'a> Sim<'a> {
         world.topology.set_link_fail_rate(a, b, value);
     }
 
+    /// Access a [`LinksIter`] to introspect inflight messages between hosts.
+    pub fn links(&self, f: impl FnOnce(LinksIter)) {
```

Comment:
Did you consider returning an iterator? Seems slightly more idiomatic

Reply:
It would certainly be nicer, but I don't believe we can since we need to borrow the world's ref cell.